README.md

```diff
@@ -236,17 +236,19 @@ export default Component;
 
 **The following environment variables are embedded during the build time.**
 
-* `process.env.RAZZLE_PUBLIC_DIR`: Path to the public directory.
+* `process.env.RAZZLE_PUBLIC_DIR`: Absolute path to the public directory in the server's filesystem.
 * `process.env.RAZZLE_ASSETS_MANIFEST`: Path to a file containing compiled asset outputs
 * `process.env.REACT_BUNDLE_PATH`: Relative path to where React will be bundled during development. Unless you are modifying the output path of your webpack config, you can safely ignore this. This path is used by `react-error-overlay` and webpack to power up the fancy runtime error iframe. For example, if you are using common chunks and an extra entry to create a vendor bundle with stuff like react, react-dom, react-router, etc. called `vendor.js`, and you've changed webpack's output to `[name].js` in development, you'd want to set this environment variable to `/static/js/vendor.js`. If you do not make this change, nothing bad will happen, you will simply not get the cool error overlay when there are runtime errors. You'll just see them in the console. Note: This does not impact production bundling.
 * `process.env.VERBOSE`: default is false, setting this to true will not clear the console when you make edits in development (useful for debugging).
-* `process.env.PORT`: default is `3000`, unless changed
-* `process.env.HOST`: default is `0.0.0.0`
+* `process.env.PORT`: The `BUILD_TARGET=server` build listens on this port for all NODE_ENVs. default is `3000`
```

Comment:
Isn’t this available on client?

Reply:
It depends what you mean by client. I'm talking about the webpack build target here. The browser connects to the PORT used by BUILD_TARGET=server for the HTML, then connects to PORT+1 for HMR. (HMR is the BUILD_TARGET=client)